app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -97,6 +98,15 @@ class ChatActivity : AppCompatActivity(), IChatView {
         chatPresenter.setUp()
         chatPresenter.checkPreferences()
         setEditText()
+        /*
+         Adding on swipe listener for swipe between @see(SkillListingFragment.kt) and ChatActivity.kt
+         */
```

Comment:
Use '//' as it is a single line comment. Again, not a big issue, but please follow the coding style used in the project.

Reply:
@arundhati24 In the previous PR, I was asked to have a multiline comment :P

Reply:
I don't have an idea which PR you are talking about. But, it would be really nice if you could use '//' for a single line comment for the changes that you have made here. Thanks :)